app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt

```diff
@@ -404,6 +404,110 @@ class SettingsViewModel(
+                        when {
+                            prefsRepo.jumpForwardSeconds > 0 -> {
+                                "${prefsRepo.jumpForwardSeconds} " + Injector.get()
+                                    .applicationContext().resources.getString(R.string.seconds)
+                            }
+                            else -> throw NoWhenBranchMatchedException()
+                        }
+                    )
+                ),
+                explanation = FormattableString.from(R.string.settings_jump_forward_explanation),
+                click = object : PreferenceClick {
+                    override fun onClick() {
+                        showOptionsMenu(
+                            options = listOf(
+                                FormattableString.from(R.string.settings_jump_10_seconds),
+                                FormattableString.from(R.string.settings_jump_15_seconds),
+                                FormattableString.from(R.string.settings_jump_20_seconds),
+                                FormattableString.from(R.string.settings_jump_30_seconds),
+                                FormattableString.from(R.string.settings_jump_60_seconds),
+                                FormattableString.from(R.string.settings_jump_90_seconds)
+                            ),
+                            title = FormattableString.from(R.string.settings_jump_forward_title),
+                            listener = object : BottomChooserItemListener() {
+                                override fun onItemClicked(formattableString: FormattableString) {
+                                    check(formattableString is FormattableString.ResourceString)
+                                    when (formattableString.stringRes) {
+                                        R.string.settings_jump_10_seconds -> prefsRepo.jumpForwardSeconds =
+                                            10L
+                                        R.string.settings_jump_15_seconds -> prefsRepo.jumpForwardSeconds =
+                                            15L
+                                        R.string.settings_jump_20_seconds -> prefsRepo.jumpForwardSeconds =
+                                            20L
+                                        R.string.settings_jump_30_seconds -> prefsRepo.jumpForwardSeconds =
+                                            30L
+                                        R.string.settings_jump_60_seconds -> prefsRepo.jumpForwardSeconds =
+                                            60L
+                                        R.string.settings_jump_90_seconds -> prefsRepo.jumpForwardSeconds =
+                                            90L
+                                        else -> throw NoWhenBranchMatchedException("Unknown item: ${formattableString.stringRes}")
+                                    }
+                                    // todo: update UI → change ImageView?
```

Comment:
I'll write up a little explanation of how to do this after I finish the review 👍, but nothing to change here, we can just observe preferences

Reply:
(in the currentlyplayingviewmodel)